package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.1",
   "description": "Kubernetes Contributor Site",
   "main": "none.js",
-  "dependencies": {},
   "devDependencies": {
-    "autoprefixer": "^9.7.6",
-    "postcss-cli": "^7.1.0"
+    "autoprefixer": "^10.4.20",
+    "postcss": "^8.4.47",
+    "postcss-cli": "^11.0.0"
```

Comment:
Aside: do we need any of these outside of dev (eg for a production Netlify build)?

Reply:
I don't think so. They have been `devDependencies` before the PR and stuff worked so I don't think its needed. In fact, `autoprefixer` and `postcss-cli` is installed globally in the container (which is probably why it works and is probably not how it should be).